src/form/time/select/TimeSelect.tsx

```diff
@@ -0,0 +1,102 @@
+import "./_time-select.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import {formatDateWithOptions} from "../../../core/utils/time/timeUtils";
+import {DATE_FORMAT} from "../../../core/utils/time/timeConstants";
+import {DropdownOption} from "../../../dropdown/list/item/DropdownListItem";
+import TimeInput from "../input/TimeInput";
+import TimeDropdown from "../dropdown/TimeDropdown";
+
+export type TimeSelectDropdownOption = DropdownOption<
+  string,
+  {date: Date; differenceInMinutes: number}
+>;
```

Comment:
```suggestion
export type TimeSelectDropdownOption = TimeDropdownOption;
```

Or, we can remove `TimeSelectDropdownOption` because I think they can just use `TimeDropdownOption`

Reply:
Right, we don't need duplicated types. I'll remove `TimeSelectDropdownOption`. Thanks!